### ISSUE TITLE
add new scaler for alibaba cloud sls product

### ISSUE THREAD

**Issue by silenceper** [kedacore/keda#1152]
We collect Ingress log to [SLS](https://www.alibabacloud.com/help/product/28958.htm) (Log Service),analysis and make up metrics such as sls_ingress_qps,sls_ingress_latency_avg and so on. The external metrics can help developers from a different angle.

I want add alibaba sls product to keda.(like `azure-log-analytics`)

**Comment by tomkerkhove** [comment]
Are you suggesting a Alibaba Log Service scaler? I'm not sure I get your question fully.

**Comment by ringtail** [comment]
@silenceper https://github.com/AliyunContainerService/alibaba-cloud-metrics-adapter/blob/master/docs/metrics/sls.md. Use external metrics adapter would help.

**Comment by tomkerkhove** [comment]
Thanks for sharing, but maybe it's nice to have this as a built-in scaler?

**Comment by ringtail** [comment]
> Thanks for sharing, but maybe it's nice to have this as a built-in scaler?

Yes, I agree. External Adapter could share the basic code to built-in scaler.

**Comment by ayoyu** [comment]
Hi @tomkerkhove I can pick up this to add the Alibaba simple log service scaler (similar to azure log analytics)  if it's still relevant and no one is working on it.

**Comment by tomkerkhove** [comment]
Sure, thank you!
